components/admin_console/billing/billing_subscriptions.tsx

```diff
@@ -142,8 +145,18 @@ const BillingSubscriptions: React.FC<Props> = () => {
                         className='BillingSubscriptions__topWrapper'
                         style={{marginTop: '20px'}}
                     >
-                        <div style={{border: '1px solid #000', width: '568px'}}>
+                        <div style={{border: '1px solid #000', width: '568px', padding: '8px', backgroundColor: '#fff'}}>
```

Comment:
Are you planning on moving these styles out to a css module or global styles at some point?

Reply:
These are temporary just so I can get an idea of the layout.
All of these in-line styles will be going away and replaced with actual components or `div`s with classes attached.